addons/mail/static/src/components/rtc_video/rtc_video.js

```diff
@@ -60,11 +60,21 @@ export class RtcVideo extends Component {
     //--------------------------------------------------------------------------
 
     /**
+     * Plays the video as some browsers may not support or block autoplay.
+     *
      * @private
      * @param {Event} ev
      */
     async _onVideoLoadedMetaData(ev) {
-        await ev.target.play();
+        try {
+            await ev.target.play()
+        } catch (error)  {
+            if (error.name != 'NotAllowedError') {
+                // ignored as some browsers may reject play() calls that do not
+                // originate from a user input.
```

Comment:
It's strange to have the comment above the "throw", even though it says the opposite, I would inverse the condition, return here and throw outside of the condition.